### DISCUSSION TITLE
Langchain Agents as Custom Tools

### DISCUSSION THREAD

**Discussion by yakigac** [langflow-ai/langflow#40] [Ideas]
Hello,
I appreciate your work! It's an excellent project.

While exploring langflow features, I noticed that it does not currently support the use of langchain Agents as Tools. In my opinion, integrating langchain agent as a tool is an important aspect for solving tasks step by step, and it could make langflow more versatile.

I understand that you may have already considered this feature. I'd be interested in hearing your thoughts on the matter and whether you believe it would be a worthwhile addition to the project. If you're open to the idea, I would be more than happy to contribute to the development process.

Looking forward to hearing from you.

**Comment by ogabrielluiz** [comment]
I completely agree with you. This is something I'm especially interested in and what made me open my eyes to langchain in the first place.

This is very high in our priorities and we'll work on it as soon as we get the basic stuff sorted(docs, bugs. etc.) 

Thanks for bringing this up, though, as this is why we opened the project to the community: to help us see what we should prioritize.

If you or someone you know is interested in helping us out you are most welcome.
